### DISCUSSION TITLE
Docs layout incorrect and potentially broken?

### DISCUSSION THREAD

**Discussion by Lord-V15** [fastapi/fastapi#11267] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
N/A
```

### Description

This issue is not related to the framework but instead the Mkdocs integration that builds the actual documentation website. Recently, I opened it and found this :
![Screenshot 2024-03-11 at 4 19 00 AM](https://github.com/tiangolo/fastapi/assets/46837998/041da9ae-eda8-47d0-a079-c97905bfdafa)
I couldn't find any related issue. Is this known? Is someone fixing it?

### Operating System

macOS

### FastAPI Version

N/A

### Pydantic Version

N/A

### Python Version

N/A

**Comment by Lord-V15** [comment]
Just to confirm I am seeing it in multiple sections :

https://github.com/tiangolo/fastapi/assets/46837998/6a87af55-a048-4a98-8d31-6ed02f097740

**Reply by JavierSanchezCastro** [reply]
Something has been wrong since the last version was released, I think that other discussion is related: https://github.com/tiangolo/fastapi/discussions/11257

**Reply by tiangolo** [reply]
Thanks @JavierSanchezCastro, but that's a different problem, related to how the API reference is rendered.

**Comment by tiangolo** [comment, marked as answer]
I tried here and everything seems normal :thinking:

Have you tried it on another browser? On an incognito window? Also on mobile?

You could try to remove any site data and cache, or hard refresh, that might solve it.

**Reply by Lord-V15** [reply]
after clearing site data and cache it worked :
<img width="633" alt="Screenshot 2024-03-13 at 12 29 21 PM" src="https://github.com/tiangolo/fastapi/assets/46837998/00d1442b-59c3-4b24-b0e7-cb305a66a8b0">
